_includes/_howto/deployment_prerequisites.md

```diff
@@ -51,25 +52,52 @@ To create a service account for Forseti Inventory, Scanner, and Enforcer, follow
       ```bash
       $ gcloud organizations add-iam-policy-binding ORGANIZATION_ID \
       --member=serviceAccount:YOUR_SERVICE_ACCOUNT \
-      --role=roles/compute.networkAdmin
+      --role=roles/compute.networkViewer
       ```
       ```bash
       $ gcloud organizations add-iam-policy-binding ORGANIZATION_ID \
       --member=serviceAccount:YOUR_SERVICE_ACCOUNT \
-      --role=roles/editor
+      --role=roles/iam.securityReviewer
       ```
       ```bash
       $ gcloud organizations add-iam-policy-binding ORGANIZATION_ID \
       --member=serviceAccount:YOUR_SERVICE_ACCOUNT \
-      --role=roles/iam.securityReviewer
+      --role=roles/appengine.appViewer
+      ```
+      ```bash
+      $ gcloud organizations add-iam-policy-binding ORGANIZATION_ID \
+      --member=serviceAccount:YOUR_SERVICE_ACCOUNT \
+      --role=roles/servicemanagement.quotaViewer
+      ```
+      ```bash
+      $ gcloud organizations add-iam-policy-binding ORGANIZATION_ID \
+      --member=serviceAccount:YOUR_SERVICE_ACCOUNT \
+      --role=roles/cloudsql.viewer
       ```
       ```bash
       $ gcloud organizations add-iam-policy-binding ORGANIZATION_ID \
       --member=serviceAccount:YOUR_SERVICE_ACCOUNT \
-      --role=roles/resourcemanager.folderAdmin
+      --role=roles/cloudsql.viewer
       ```
       ```bash
```

Comment:
Should we call this out as optional, and if they aren't using enforcer to not include it?